### ISSUE TITLE
No Moto CloudFormation support for AWS::KinesisFirehose::DeliveryStream / AWS::S3::BucketPolicy

### ISSUE THREAD

**Issue by GuzzoLM** [localstack/localstack#1616]
I'm trying to build an infrastructure with kinesis firehose and s3 buckets, using the following template.

```json
{
    "AWSTemplateFormatVersion": "2010-09-09",
    "Resources": {
        "MyDeliveryStream": {
            "Type": "AWS::KinesisFirehose::DeliveryStream",
            "Properties": {
                "DeliveryStreamName": "MyDeliveryStream",
                "DeliveryStreamType": "DirectPut",
                "S3DestinationConfiguration": {
                    "BucketARN": "aws:aws:s3:::MyBucket",
                    "BufferingHints": {
                        "IntervalInSeconds": 60,
                        "SizeInMBs": 1
                    },
                    "CompressionFormat": "UNCOMPRESSED",
                    "RoleARN": "arn:aws:s3:::BucketPolicy:MyBucketPolicy"
                }
            },
            "DependsOn": [
                "MyBucket"
            ]
        },
        "MyBucket": {
            "Type": "AWS::S3::Bucket",
            "Properties": {
                "BucketName": "MyBucket"
            }
        },
        "MyBucketPolicy": {
            "Type": "AWS::S3::BucketPolicy",
            "Properties": {
                "Bucket": {
                    "Ref": "MyBucket"
                },
                "PolicyDocument": {
                    "Statement": [
                        {
                            "Effect": "Allow",
                            "Action": [
                                "s3:AbortMultipartUpload",
                                "s3:GetBucketLocation",
                                "s3:GetObject",
                                "s3:ListBucket",
                                "s3:ListBucketMultipartUploads",
                                "s3:PutObject"
                            ],
                            "Resource": [
                                "arn:aws:s3:::MyBucket",
                                "arn:aws:s3:::MyBucket/*"
                            ]
                        }
                    ]
                }
            }
        }
    }
}
```
I'm generating the infrstructure with the command
`aws --endpoint-url=http://localhost:4581 cloudformation create-stack --stack-name localstack --template-body file:///tmp/localstack-cloudformation.json`

The bucket is created without problems, but the BucketPolicy and the Firehose delivery stream are returning errors:

`No Moto CloudFormation support for AWS::S3::BucketPolicy`
`No Moto CloudFormation support for AWS::KinesisFirehose::DeliveryStream`

I'm running localstack on docker. The `SERVICES` are **s3,firehose,cloudformation**
How can I fix this?

**Comment by pavan2106** [comment]
Hi,
I am also getting the similar error for  **AWS::Events::Rule**
message in the log: No Moto CloudFormation support for AWS::Events::Rule
please suggest, if there is any solution around to deploy AWS::Events::Rule

**Comment by whummer** [comment]
Thanks for reporting @GuzzoLM @pavan2106 . This should be fixed in #2301 - can you please give it a try with the `latest` version? Please report here if the problem persists. Thanks
